app/javascript/packages/document-capture/services/upload.js

```diff
@@ -18,13 +18,14 @@ export function toFormData(object) {
 /**
  * @type {import('../context/upload').UploadImplementation}
  */
-async function upload(payload, { endpoint, csrf }) {
+async function upload(payload, { endpoint, csrf, sessionUUID }) {
+  const payloadWithSessionUUID = { ...payload, document_capture_session_uuid: sessionUUID };
```

Comment:
I think that `document_capture_session_uuid` should just be inside of payload

Reply:
I'm going to move this to a different PR